external/netfx-conflicts/netfx-conflicts.depproj

```diff
@@ -9,6 +9,7 @@
     <PackageTargetFallback>netstandard1.1;netstandard1.2;netstandard1.3</PackageTargetFallback>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
+    <BinPlaceNETFXRuntime>false</BinPlaceNETFXRuntime>
```

Comment:
Yes we currently have an outerloop run and it wouldn't fail it would pass because Jenkins wouldn't find any test results and they would mark the build as passed. See: https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_release/19/